packaging/nnstreamer.spec

```diff
@@ -1134,6 +1134,7 @@ cp -r result %{buildroot}%{_datadir}/nnstreamer/unittest/
 %files single-devel
 %{_includedir}/nnstreamer/tensor_typedef.h
 %{_includedir}/nnstreamer/nnstreamer_plugin_api_filter.h
+%{_includedir}/nnstreamer/nnstreamer_plugin_api_trainer.h
```

Comment:
Is it necessary to add this header to the RPM at this moment? IMO, at least, packaging should be handled as a separate commit.

Reply:
Thank you.
I will separate

Reply:
Done, thank you ([03e3085](https://github.com/nnstreamer/nnstreamer/pull/3977/commits/03e30851427b383ca6f5b7bb0df7ac090a5a1869))